ios/Exponent/Versioned/Core/EXVersionManager.m

```diff
@@ -108,7 +107,8 @@ - (void)bridgeWillStartLoading:(id)bridge
   // Keep in mind that it is possible this will return a EXDisabledRedBox
   RCTRedBox *redBox = [self _moduleInstanceForBridge:bridge named:@"RedBox"];
   [redBox setOverrideReloadAction:^{
-    [[EXKernel sharedInstance] reloadVisibleApp];
+      [[NSNotificationCenter defaultCenter]
+     postNotificationName:@"EXReloadActiveAppRequest" object:nil];
```

Comment:
This way the notification name will be versioned as well, so the kernel wouldn't receive it. Use `EX_UNVERSIONED` macro to let the versioning script know this string must be unversioned.
```suggestion
     postNotificationName:EX_UNVERSIONED(@"EXReloadActiveAppRequest") object:nil];
```